setup.py

```diff
@@ -112,6 +112,10 @@
     "cassandra-driver>=3.24.0,<4",
 ]
 
+SCYLLADB_REQUIRED = [
```

Comment:
Update your branch. All python specific changes should be in master by now.

Reply:
Done